Frameworks/CoreText/CTLine.mm

```diff
@@ -278,21 +303,24 @@ void CTLineDraw(CTLineRef lineRef, CGContextRef ctx) {
  @Status Interoperable
 */
 CFIndex CTLineGetGlyphCount(CTLineRef line) {
-    return line ? static_cast<_CTLine*>(line)->_glyphCount : 0;
+    RETURN_RESULT_IF_NULL(line, 0);
+    return line->_glyphCount;
```

Comment:
I mean in the paradigm this was done in CTLineGetGlyphCount _was_ the getter, but now since we're updating this to use CppBase and touching everything we should use better style 🛩